finality-aleph/src/sync/handler.rs

```diff
@@ -684,7 +785,7 @@ mod tests {
     fn handles_request() {
         let (mut handler, backend, _keep) = setup();
         let initial_state = handler.state().expect("state works");
-        let peer = rand::random();
+        let peer: MockPeerId = rand::random();
```

Comment:
Huh, you now need this typing?

Reply:
it's a mystery

Reply:
Meaning?

Reply:
ask the compiler...

Reply:
welp, turns out the compiler says it's ok, my bad, fixed